_templates/_page_openshift.html.erb

```diff
@@ -67,12 +58,32 @@
 
       <span>
         <div class="alert alert-danger" role="alert" id="support-alert">
-          <strong>You are viewing documentation for a release that is no longer supported.</strong> The latest supported version of version 3 is <a href="https://docs.openshift.com/container-platform/3.11/welcome/index.html" class="link-primary" style="color: #545454 !important;">[3.11]</a>. For the most recent version 4, see <a href="https://docs.openshift.com/container-platform/latest/welcome/index.html" style="color: #545454 !important" class="link-primary">[4]</a>.
+          <strong>You are viewing documentation for a release that is no longer maintained.</strong> The latest supported version of version 3 is <a href="https://docs.openshift.com/container-platform/3.11/welcome/index.html" class="link-primary" style="color: #545454 !important;">[3.11]</a>. For the most recent version 4, see <a href="https://docs.openshift.com/container-platform/latest/welcome/index.html" style="color: #545454 !important" class="link-primary">[4]</a>.
         </div>
       </span>
 
       <% end %>
 
+      <% if ((unsupported_versions.include? version) && (distro_key == "openshift-acs")) %>
```

Comment:
Same question as above - this is not part of the PDF updates.

Reply:
`_templates/_page_openshift.html.erb` I copied this file from the `main` branch.